### PR TITLE
chore: halve the number of ci runs

### PR DESCRIPTION
**Pull request by rgrinberg**
Signed-off-by: Rudi Grinberg <me@rgrinberg.com>

### CODE REVIEW

**Comment by Alizter** [comment]
This will mean it won't run on peoples forks when they push to a branch. But I guess it can be manually dispatched instead.

**Review by Alizter — APPROVED** [review]

**Comment by rgrinberg** [comment]
Hmm, this doesn't seem right. There's no CI on this PR.

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 name: CI
 
 on:
-  - push
-  - pull_request
+  pull_request:
```
Comment (Alizter) [inline .github/workflows/workflow.yml:4]: I think you made the condition for pull requests empty.

Comment (Alizter) [reply .github/workflows/workflow.yml:4]: (remove the colon)

Comment (rgrinberg) [reply .github/workflows/workflow.yml]: Didn't make a difference. Anyway, I copied it from ocamllsp and it seems to work there.

**Comment by rgrinberg** [comment]
Done by Rafal

**Pull request closed without merging**